meeshkan/serve/mock/views.py

```diff
@@ -86,8 +86,8 @@ def _serve(self):
         )
 
         logger.debug(request)
-        initial_response = self.response_matcher.get_response(request)
-        response = self.callback(request, initial_response)
+        response = self.response_matcher.get_response(request)
```

Comment:
This `get_response` sounds like a pure function but as far as I know it has all kinds of side-effects. I wonder if those side-effects could be somehow made explicit here either by renaming the method or splitting it into pure and non-pure steps.

Reply:
It was a pure function. I made it RequestProcessor and process to not be confused anymore.